exe/niftany

```diff
@@ -10,7 +10,7 @@ require 'scss_lint/cli'
 require 'colorize'
 
 def parse_opts(args)
-  options = OpenStruct.new
+  options = Struct.new
```

Comment:
@whereismyjetpack Getting this error here when I try to run niftany:

`13:in `new': wrong number of arguments (given 0, expected 1+) (ArgumentError)`

Is OpenStruct deprecated?

Reply:
It works with `OpenStruct.new`